models/lido_liquidity/kyberswap_ethereum/lido_liquidity_kyberswap_ethereum_schema.yml

```
@@ -0,0 +1,76 @@
+version: 2
+
+models:
+  - name: lido_liquidity_kyberswap_ethereum
+    meta:
+      blockchain: ethereum
+      project: lido
+      contributors: gregshestakovlido, ppclunghe
+    config:
+      tags: ['ethereum','lido','liquidity']
+    description: >
+        Lido wstETH liquidity pools on Kyberswap Ethereum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - pool_name
+            - pool
+            - chain
+            - project
+            - fee
+            - time
+            - main_token
+            - main_token_symbol
+            - paired_token
+            - paired_token_symbol
+            - main_token_reserve
+            - paired_token_reserve
+            - main_token_usd_reserve
+            - paired_token_usd_reserve
+            - trading_volume
```

Hi @gregshestakovlido,
do you have the idea for the minimum combination of columns to be the unique_key?

eg.
```suggestion
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - pool
            - chain
            - project
            - fee
            - time
```

and we may rename `chain` to `blockchain` to be consistent with other models.

---

Done